### PR TITLE
multitail: add page

### PR DESCRIPTION
**Pull request by larshb**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/multitail.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# multitail
+
+> Extension of tail.
```
Comment (sbrl) [inline pages/common/multitail.md]: Is there a more information link we can include directly below this line? (example page that includes this syntax: `git`)

File: pages/common/multitail.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Automatically add new files to a window:
+
+`multitail -Q {{pattern}}`
+
+- Show 5 logfiles while merging 2 and put them in 2 columns with only one in the left column:
+
+`multitail -s 2 -sn 1,3 {{mergefile}} -I {{file1}} {{file2}} {{file3}} {{file4}}`
+
+- More examples:
```
Comment (sbrl) [inline pages/common/multitail.md]: Ah, I see. The link in this example should be the one in the more information I've mentioned in the comment above I think.

File: pages/common/multitail.md (hunk 1 above)
Comment (larshb) [reply pages/common/multitail.md]: I moved the link at the bottom. Is it ok to point directly to the examples sub-page?

Comment (sbrl) [reply pages/common/multitail.md]: Fine by me!

File: pages/common/multitail.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Tail all files in a directory in a single stream:
+
+`multitail -Q 1 '{{directory}}/*'`
+
+- Automatically add new files to a window:
+
+`multitail -Q {{pattern}}`
+
+- Show 5 logfiles while merging 2 and put them in 2 columns with only one in the left column:
```
Comment (agnivade) [inline pages/common/multitail.md]: This is lifted verbatim from the home page and could be improved IMO. It is not clear to me what this is trying to do. So first of all, it shows 5 files, which means the 4 input file and the output file. And the merged file is the combination of 2 files ? Which 2 ? What does putting the output in 2 columns mean ? Which one is in the left column ?

Sorry but I don't understand any of it.

File: pages/common/multitail.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`multitail -s 2 -sn 1,3 {{mergefile}} -I {{file1}} {{file2}} {{file3}} {{file4}}`
```
Comment (agnivade) [inline pages/common/multitail.md]: 2 and 1,3 should be in tokens (`{{}}`)

File: pages/common/multitail.md (hunk 3 above)
Comment (sbrl) [reply pages/common/multitail.md]: Ah, I must have missed this! We should open a new PR to improve this.

**Pull request merged**